student-store-ui/src/components/Home/Home.jsx

```diff
@@ -1,10 +1,53 @@
-import * as React from "react"
-import "./Home.css"
+import React from "react";
+import "./Home.css";
+import Hero from "../Hero/Hero";
+import Subnavbar from "../Subnavbar/Subnavbar";
+import Footer from "../Footer/Footer";
+import About from "../About/About";
+import Contact from "../Contact/Contact";
+import {useState } from "react"
+import ProductGrid from "../ProductGrid/ProductGrid";
 
-export default function Home() {
+export default function Home({ products, setProducts, shoppingCart, setShoppingCart, handleAddQuantity, handleSubstractQuantity, getQuantity}) {
```

Comment:
format this using multiple lines (best practice is limit 80-characters per line)